spec/rails_helper.rb

```diff
@@ -6,7 +6,9 @@
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
 require 'simplecov'
-SimpleCov.start
+SimpleCov.start do
+  add_filter "app/controllers/graphql_controller.rb"
```

Comment:
Good thinking.

Reply:
Nice Ive been wondering how to get simplecov to ignore things!